src/scripts/constants/ClapprEvents.js

```diff
@@ -0,0 +1,580 @@
+const Events = {}
```

Comment:
Currently, the distributed version of `clappr` doesn't allow you to import their `events.js` module directly. Instead, you must import it from the entire clappr bundle, meaning if we want to just access the clappr events constants we need to import the entire bundle. 

This file is just a copy of their `events.js`, and is a temporary workaround IMO. But it helps w/ the bundle size by a good amount (will report statistics a little later). 

I plan on opening an issue (and perhaps PR'ing) clappr such that they distribute the events module directly so that it can be imported as a standalone.

Reply:
I've opened [a PR](https://github.com/clappr/clappr/pull/1640) w/ `clappr` so that we can directly import these constants instead of redefining them.